apps/meteor/client/components/message/content/attachments/DefaultAttachment.tsx

```diff
@@ -82,7 +82,7 @@ const DefaultAttachment = (attachment: DefaultAttachmentProps): ReactElement =>
 										return field;
 									}
 
-									const { value, title, ...rest } = field;
+									const { value = '', title = '', ...rest } = field;
 
 									return {
```

Comment:
Should we be returning this if `value` or `title` is `''`?

Maybe we could avoid sending an empty `<MarkdownText>` component if the respective property is not defined

Reply:
Makes sense, done!